Rome-Blockchain-Labs/terminal-widgets
# RT-1092 Redesign Widget's Wallet Selection Modal (Univ2 and Velox)

## Description
- Update Wallet Modal styling for velox and univ2 widgets

Test
- run romefrontend locally
- run terminal-widgets/trades locally
- try opening univ2  clones and velox and check that wallet modal sstyling is updated
